products/looker.md

```diff
@@ -13,9 +13,36 @@ releaseColumn: false
 releaseDateColumn: true
 releaseImage: https://cloud.google.com/static/looker/docs/images/2022-std-supp-releases.png
 releases:
+-   releaseCycle: "22.20"
+    releaseDate: 2022-11-14
```

Comment:
we are not adding any RC or possible release before its release
so could you remove this one

Reply:
we can add this one after its release within another pr

Reply:
Version removed.